docs/src/content/docs/components/data-services/lake-formation.mdx

```diff
@@ -0,0 +1,426 @@
+                ...
+              },
+              ]
+          },
+       },
+       ...
+     }
+    });
+    ```
+  </Fragment>
+  <Fragment slot="python">
+    ```python
+    #!/usr/bin/env python3
+    import aws_cdk as cdk
+    import aws_data_landing_zone as dlz
+
+    app = cdk.App()
+    dlz.DataLandingZone(app,
+        organization=dlz.DLzOrganization(
+            ous=dlz.OrgOus(
+                workloads=dlz.OrgOuWorkloads(
+                    accounts=[
+                        dlz.DLzAccount(
+                            name='development',
+                            account_id='111111111111',
+                            lake_formation=[dlz.DlzLakeFormationProps(
+                                region=dlz.Region.EU_WEST_1,
+                                admins=['arn:aws:iam::123456789012:role/Admin'],
+                                tags=[ ... ],
+                                permissions=[ ... ]
+                            )]
+                        ),
+                        dlz.DLzAccount(
+                            name='production',
+                            account_id='222222222222',
+                        ),
+                    ],
+                ),
+            )
+        ),
+    )
+````
+  </Fragment>
+</DualCode>
+
+## LF-Tags and Permissions
+
+The `tags` array defines the tags created in Lake Formation. Each tag consists of a key and a list of possible values.
+
+To manage permissions on these tags, you use the `share` property, which allows you to share tags either within the account
+or with external accounts. If you want to grant permissions on only a subset of tag values, use the `specificValues`
+property. If this property is not provided, permissions will be granted to all values of the tag by default.
+
+### Sharing Tags Within an Account
+
+The `share.withinAccount` property allows you to share tags within the same account. You specify the IAM principals
+(granted permissions on the tag) in the `principals` property. These principals can be IAM roles, IAM users, SAML users,
+or SAML groups.
+
+By default, the CDK role is assigned as a Lake Formation admin. Normally, Lake Formation admins cannot manage tags created
+by other admins. To address this, we explicitly grant all admins access to the tags created by the CDK DLZ construct. This
+is equivalent to adding every admin under the `share.withinAccount.principals` property with full access.
+
+### Sharing Tags with External Accounts
+
+When sharing tags with external accounts, the `share.withExternalAccount` property is used. The primary difference
+between sharing tags internally (`withinAccount`) and externally (`withExternalAccount`) is that we restrict certain
+actions externally. Specifically, the `TagAction.ALTER` and `TagAction.DROP` actions are not allowed for external sharing,
+which ensures that external sharing remains read-only. This restriction prevents CloudFormation errors later in the process.
```

Comment:
it's not like *WE* resctrict that. It's literally not allowed by LakeFormation itself.

Reply:
That is what the last sentence implies, we are restricting it in code as CDK itself will allow it and then it will only error during CloudFormation deployment.